sql/src/test/java/org/apache/druid/sql/calcite/expression/ExpressionsTest.java

```diff
@@ -1072,6 +1075,108 @@ public void testPad()
     );
   }
 
+  @Test
+  public void testContains()
+  {
```

Comment:
You should include tests here using `testHelper.testExpression` too. It'll test that `toDruidExpression` works properly, which will be used if someone uses this operator outside of a leaf filter.

Reply:
will add those as I add corresponding native functions.